docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -77,7 +40,7 @@ Let's create two simple Maestro flows for the example app. Start by creating a d
 
 Inside, create a new file called **home.yml**. This flow will launch the app and assert that the text "Welcome!" is visible on the home screen.
 
-```yaml maestro/home.yml
+```yaml .maestro/home.yml
```

Comment:
Since the directory name changed here from "maestro" to ".maestro", let's update the same on [line 39](https://github.com/expo/expo/pull/35826/files#diff-843254adf809bac8c08e13a8bf71989112c1e2ff96690e5b911de24599c349c4R39).